gateway/utils/graphql/read.go

```diff
@@ -174,7 +199,16 @@ func (graph *Module) execPreparedQueryRequest(ctx context.Context, field *ast.Fi
 	}
 
 	go func() {
-		result, err := graph.crud.ExecPreparedQuery(ctx, dbAlias, id, &req, reqParams)
+		result, metaData, err := graph.crud.ExecPreparedQuery(ctx, dbAlias, id, &req, reqParams)
+		if err != nil {
+			cb("", "", nil, err)
+			return
+		}
+
+		if req.Debug {
+			val := store["_query"]
+			val.(*utils.Array).Append(metaData)
```

Comment:
I suggest you let metaData be a proper struct for documentation purposes. Convert the struct to map here using structs.Map